website/docs/plugins/rq_launcher.md

```diff
@@ -68,6 +70,12 @@ export REDIS_DB="0"
 export REDIS_PASSWORD=""
 ```
 
+SSL can be enabled like so:
+```commandline
+export REDIS_SSL="True"
+export REDIS_SSL_CA_CERTS="/etc/ssl/certs/ca-certificates.crt"
+```
+
```

Comment:
```suggestion
Enable SSL by setting the relevant environment variables. e.g:
```commandline
export REDIS_SSL=true
export REDIS_SSL_CA_CERTS=/etc/ssl/certs/ca-certificates.crt
```

```

(Note that I removed the quotes and lower-cased the true).

Reply:
✅